packages/atomic/src/components/search-box-suggestions/atomic-search-box-recent-queries/atomic-search-box-recent-queries.tsx

```diff
@@ -141,12 +141,13 @@ export class AtomicSearchBoxRecentQueries {
       key: `recent-${value}`,
       query: value,
       content: (
-        <div class="flex items-center">
-          <atomic-icon icon={Clock} class="w-5 h-5 mr-2 -ml-1"></atomic-icon>
+        <div class="flex items-center break-all">
```

Comment:
I was gonna comment on using `break-word` instead, but yeah, google uses break-all also. Which is a bit weird... 

But it's true that if we have one suggestions that's just one massively long word, we'd have problems.

So 👍